cmd/git-go-patch/rebase.go

```diff
@@ -27,28 +29,50 @@ You can use this command to apply fixup and squash commits generated by the "git
 "--fixup" and "--squash". To do this, configure Git using "git config --global rebase.autoSquash 1"
 before running this command.
 
+With the -shell flag, this opens an interactive terminal session in the submodule directory instead
+of running git rebase directly. When you exit the terminal, it automatically runs 'git go-patch extract'
```

Comment:
Why instead? I don't think it makes sense to put this code in `rebase` if it doesn't start the `rebase`.

Reply:
agreed, I've configured it to actually trigger the rebase